### ISSUE TITLE
Hide (don't show) destinations that are never of interest

### ISSUE THREAD

**Issue by kiwidude** [RequestPolicy/requestpolicy#389]
Some destinations are never ever of any interest and a 100% waste of bandwidth (and privacy), like clickjunk.net or uselessanalytics,com. They also waste time and space in the UI.
It would be very useful to hide such destinations altogether from the "Blocked destinations" list (when clicking on the red flag for blocking/unblocking), and to also not show the flag red when there are only hidden destinations blocked.
Currently my flag is always red because I never have a use for having uselessanalytics.com etc. waste my time, and there are precious few sites without this junk. The colour of the flag has therefore no useful meaning and conveys no relevant information.

(Now to work out how to set labels for issues in github...)

**Comment by Cathryne** [comment]
Agreed! But I think this has already been suggested: https://github.com/RequestPolicy/requestpolicy/issues/368 ;-)

**Comment by kiwidude** [comment]
Thanks Cathryne. I'm by no means a computer newbie, but the search field for issues is still absent, and sorry I'm not browsing through the lot. Labelling issues is also elusive. Apologies for the duplicate. Can't change status either - never used such a rubbishy bug tracker before.

**Comment by Cathryne** [comment]
No problem ;-)   The general "This repository" search box up top works for both code & discussions.
